doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -123,60 +120,49 @@ emitters:
 - `clear-output-folder` indicate if you want to clear up the output folder.
```

Comment:
Please remove references to the .NET Core 3.1 SDK on L36; it reached end-of-life on December 13th and is no longer supported.

Reply:
resolved. Thanks